Integrations/SignalSciences/SignalSciences.py

```diff
@@ -389,17 +400,74 @@ def add_ip_to_whitelist_or_blacklist(url, ip, note, expires=None):
     return res
 
 
+def return_date_closer_to_now(first_date, second_date):
+    if first_date < second_date:
+        return second_date
+    return first_date
+
+
+def get_all_sites_in_corp():
+    get_sites_request_response = get_sites()
+    data_of_sites_in_corp = get_sites_request_response.get('data', [])
+    return data_of_sites_in_corp
+
+
+def get_list_of_all_site_names_in_corp():
+    data_of_sites_in_corp = get_all_sites_in_corp()
+    list_of_all_sites_names_in_corp = []
+    for site_data in data_of_sites_in_corp:
+        site_name = site_data['name']
+        list_of_all_sites_names_in_corp.append(site_name)
+    return list_of_all_sites_names_in_corp
+
+
+def get_list_of_site_names_to_fetch():
+    list_of_site_names_to_fetch = None
+    if SITES_TO_FETCH is not None:
+        list_of_site_names_to_fetch = SITES_TO_FETCH.split(',')
+    else:
+        list_of_site_names_to_fetch = get_list_of_all_site_names_in_corp()
+    return list_of_site_names_to_fetch
+
+
+def remove_milliseconds_from_iso(date_in_iso_format):
+    date_parts_arr = date_in_iso_format.split('.')
+    date_in_iso_without_milliseconds = date_parts_arr[0]
+    return date_in_iso_without_milliseconds
+
+
+def fetch_events_from_given_sites(list_of_site_names_to_fetch, desired_from_time_in_posix):
+    events_from_given_sites = []
+    for site_name in list_of_site_names_to_fetch:
+        fetch_from_site_response_json = fetch_events(siteName=site_name, from_time=desired_from_time_in_posix)
+
+        events_fetched_from_site = fetch_from_site_response_json.get('data', [])
+        events_from_given_sites.extend(events_fetched_from_site)
+    return events_from_given_sites
+
+
+def convert_python_date_to_unix_millisecond(python_date_object):
```

Comment:
remove this func and add this functionallity to datetime_to_posix_without_milliseconds